src/domain/occurrence/__tests__/CreateOccurrencePage.test.tsx

```diff
@@ -395,6 +397,9 @@ test('today is valid event start day', async () => {
   fireEvent.blur(dateInput);
   await waitFor(() => {
     expect(dateInput).toBeValid();
+    expect(dateInput).not.toHaveAttribute('aria-describedby');
+    expect(
+      screen.queryByText(messages.form.validation.date.mustNotInThePast)
+    ).not.toBeInTheDocument();
```

Comment:
This is pretty minor but I don't recommend putting many assertions into waitFor.  You could only wait for dateInput to be valid and then leave rest of the assertions out of the waitFor (below). If 1 of those assertions would fail, the you have to always wait for the timeout to give you the error but there is no reason to wait for those after the first assertion.